chainio/constructor/constructor.go

```diff
@@ -34,6 +34,7 @@ type Clients struct {
 	ElChainReader          elcontracts.ELReader
 	ElChainSubscriber      elcontracts.ELSubscriber
 	ElChainWriter          elcontracts.ELWriter
+	// TODO: should we return instrumented clients as well?
```

Comment:
umm maybe a separate constructor?

Reply:
Oh like a bare bone clients constructor (also wouldn't return metrics/reg), and then a instrumentedConstructor that returns instrumented clients and metrics/reg. I think that's a good idea, I'll update the comment. Don't want to implement right now though.